LobbyServer2/LobbyServer/LobbyServerProtocol.cs

```diff
@@ -429,6 +429,18 @@ public void HandlePlayerInfoUpdateRequest(PlayerInfoUpdateRequest request)
                 OriginalPlayerInfoUpdate = update,
                 ResponseId = request.RequestId
             };
+
+            if (CurrentServer != null)
+            {
+                CurrentServer.SendGameInfoNotifications();
+                if (CurrentServer.GameInfo.GameStatus == GameStatus.FreelancerSelecting)
+                {
+                    Send(new ForcedCharacterChangeFromServerNotification()
+                    {
+                        ChararacterInfo = playerInfo.CharacterInfo,
+                    });
+                }
+            }
```

Comment:
This looks a bit suspicious. Does the client only request player info update in FreelancerSelecting state when the player has selected fill or duplicate lancer?

Reply:
Then they can't change at this state only people who have duplicate char or wilfill can change you can't unready at this stage , and clicking another char won't change it unless we tell char change this, this is only for changing character,if this is not there they won't see what char they select, nor teammbers